qtgui/settingutil.h

```diff
@@ -0,0 +1,23 @@
+#ifndef SETTINGUTIL_H
```

Comment:
```suggestion
#ifndef QTGUI_SETTINGUTIL_H_
```
https://google.github.io/styleguide/cppguide.html#The__define_Guard